src/MgvLib.sol

```diff
@@ -77,26 +77,47 @@ contract HasMgvEvents {
   event SetGasprice(uint value);
 
   /* Market order execution */
-  event OrderStart();
-  event OrderComplete(
+  event OrderStart(); // OrderStart does not need any info, we emit all relevant data in OrderComplete or SnipesComplete
+    // tried to add all relevant data for an order.
+  // same
```

Comment:
I assume you clean up the comments before completing the PR :)

Reply:
Yes